data/assets/scene/solarsystem/planets/earth/eclipses/paths/total/2028-07-22.asset

```diff
@@ -0,0 +1,36 @@
+local earth = asset.require('scene/solarsystem/planets/earth/earth')
+local data = asset.require('../total_data')
+
+
+
+local Layer = {
+  Identifier = "Total-2028-07-22",
+  Enabled = false,
+  File = data.path .. "2028-07-22.geojson",
+  HeightOffset = 2000.0,
+  DefaultProperties = {
+    LineWidth = 2.0,
+    Extrude = true
+  }
+}
+
+
+asset.onInitialize(function()
+  -- @TODO:  The file contains a non-simple geometry which is currently not supported
+  -- openspace.globebrowsing.addGeoJson(earth.Earth.Identifier, Layer)
```

Comment:
Should we maybe display a warning message here using `openspace.PrintWarning`?  So that the user is not confused when e.g. drag-n-dropping or including the asset